WISHLIST.txt

```diff
@@ -14,4 +14,21 @@
 - craftable, upgradeable sentry bot. floats behind the player and shoots stuff
     - better batteries increase HP? or shields?
     - switchable weapon
-    - inventory/storage?
\ No newline at end of file
+    - inventory/storage?
+    
+    *AugiteSoul's ideas, edit them whenever you want ^^*
+- computercraft compat w/ MS peripherals, and auto-transit using those
+- tractor beams, forcingly move a ship, uses lots of RF
+- wide controllers, those can control machines you own in a large radius(disable all air locks?)
+- better GC boss defeat sound, death cry looks odd w/ some bosses
+- constricting attack for mummy?
+- bodies for amun?
+- basalt chest, more resistant, and is accessible only by the one who placed it 1st
+- mission MS, automatic, can get gases from atmosphere of bodies, or ores from bodies (asteroid belts mainly)

```

Comment:
Yeah. For asteroids, was on: get ores from here, but mainly stone, for gas giants, get atmospheric gases in canisters, and for other bodies, a mix from the 2. Like get some ores, and some gases. ^^